scripts/hostapd/dnsmasq.conf

```diff
@@ -1,2 +1,5 @@
 interface=wlan0
+listen-address=10.0.0.1
+bind-interfaces
+server=208.67.222.222
```

Comment:
@rTreutlein Is it necessary to hardcode OpenDNS in here? This is what fixed the issue with your client waiting for an IP forever? Or was it the previous two lines?